styles/StreamerAvatar.module.css

```diff
@@ -71,3 +71,76 @@
   font-weight: var(--font-weight-bold);
   box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
 }
+
+.visuallyHidden {
+  border: 0;
+  padding: 0;
+  margin: 0;
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px);
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  white-space: nowrap;
+}
+
+.banner__namePath {
+  font-family:  'Ribeye', sans-serif;
```

Comment:
I can't believe there are two spaces after the `:` 😂

Reply:
Pickyness OVER900. 😂  

Didn't even see that I accidentally had another space in there. :D